### PR TITLE
Add member_of_organization method to User model

### PR DESCRIPTION
**Pull request by ajspotts**
### Description
The User model was lacking a method to check if a user was a member of a particular organization.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -288,6 +288,10 @@ def selectable_organizations
     orgs
   end
 
+  def member_of_organization?(org)
+    organizations.pluck(:id).include?(org&.id)
```
Comment (pkarman) [inline app/models/user.rb]: is the `id` comparison necessary? I would have thought this would also work:

`organizations.include?(org)`

Comment (ajspotts) [reply app/models/user.rb]: it does! thanks for the catch, refactoring

**Comment by hschallhorn** [comment]
Quick Q! What's the motivation behind using `user.member_of_organization?(org)` over `org.can_access?(user)`

**Comment by ajspotts** [comment]
> Quick Q! What's the motivation behind using `user.member_of_organization?(org)` over `org.user_has_access?(user)`

the idea to write that method came about when Kat, Yoom & I were pairing during my SCM ticket and at some point there was a determination made that there seemed to me a need for that function on the User model.  That said if you find it to be redundant I’m happy to remove the change.
Also, apologies for not having some more specific context for you.  We also decided at the time that adding the function was outside the scope of the issue so I punted it until after the completion of that ticket.

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -349,6 +349,26 @@
     end
   end
 
+  context "#member_of_organization?" do
+    let(:org) { create(:organization) }
```
Comment (yoomlam) [inline spec/models/user_spec.rb]: How about adding another test for the case where `org=nil` ?

Comment (ajspotts) [reply spec/models/user_spec.rb]: added!
